src/stacked_borrows/mod.rs

```diff
@@ -79,7 +79,7 @@ pub struct Stacks {
     /// Stores past operations on this allocation
     history: AllocHistory,
     /// The set of tags that have been exposed inside this allocation.
-    exposed_tags: FxHashSet<SbTag>,
+    pub exposed_tags: FxHashSet<SbTag>,
```

Comment:
Why does this need to be `pub`? I'd expect a visitor impl somewhere inside `src/stacked_borrows`.